Description: TERMCAP_BUF is used in place of TERMCAP_BUFSIZE.
Origin: 30c9a2a6c190647e35d1ce7b4758de9081f06d10
Author: Victor Dmitriev <mrvvitek@gmail.com>
Date: Mon, 27 Apr 2020 18:12:56 +0200
Bug: https://savannah.gnu.org/bugs/?58258

---
 src/termcap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/termcap.c
+++ b/termcap.c
@@ -1066,7 +1066,7 @@
 	{
 	  if (i >= T_KEYPAD)	/* don't put keypad codes in TERMCAP */
 	    continue;		/* - makes it too big */
-#if (TERMCAP_BUF < 1024)
+#if (TERMCAP_BUFSIZE < 1024)
           if (i >= T_FEXTRA && i < T_BACKTAB) /* also skip extra vt220 keys */
             continue;
           if (i > T_BACKTAB && i < T_NAVIGATE) /* more vt220 keys */
